### ISSUE TITLE
Adding auto-comment.yml file

### ISSUE THREAD

**Issue by Anshika-Jain02** [Counselllor/Counsellor-Web#177]
*We need to add an auto-comment.yml that will automatically welcome, greet and provide necessary information to the contributors when they create new issues, PR, and when these will get assigned or merged respectively. This will promote motivation and performance*

@thestarsahil Please assign me this issue under gssoc'23

**Comment by Nishitbaria** [comment]
@Anshika-Jain02  Why are making my issue duplicate  I have already created issue: https://github.com/Counselllor/Counsellor-Web/issues/126 hey project admin @thestarsahil  please checkout this  you have already assigned me issue

**Comment by Anshika-Jain02** [comment]
@thestarsahil sir, i didnot know that it was previously assigned and i also searched for the issue but it didnot appear in the open and closed issues and sir, i have created the PR also for this so, please let me know what to do now

**Comment by Nishitbaria** [comment]
actually, I created a Pull request before you but my fork repo is deleted that wise My pull request is closed automatically I request to close your pull request

**Comment by Nishitbaria** [comment]
If I am wrong you can check out the date and also how your code and my code become the same ??  :https://github.com/Counselllor/Counsellor-Web/pull/172

**Comment by Nishitbaria** [comment]
I request to project admins solve this issue  @thestarsahil

**Comment by Anshika-Jain02** [comment]
> If I am wrong you can check out the date and also how your code and my code become the same ?? :#172

Yes, you have raised the issue before mine and i have no doubt about that.  Also, i have taken help of the internet ( maybe from the same source as yours ) to write the code for auto-comment.yml and we can't customise the code for these types of files . There are some set rules and variables that needs to be defined in the files and i used the same reference. So, two people can have the same code

**Comment by Anshika-Jain02** [comment]
@Nishitbaria Also, i am ok with whatever @thestarsahil think is best to do in this case. I am just saying that i was not aware that you have created this issue and hence, i have created my PR.

**Comment by thestarsahil** [comment]
@Nishitbaria You had Closed the PR by yourself
So it's means that someone else have to work on this 
If you haven't closed yet the credit of this PR will be your's 
But as for now @Anshika-Jain02 is now handling this 
Thanks @Nishitbaria Please look to other issues and have patience, We need time to review PR !

**Comment by Nishitbaria** [comment]
**@Nishitbaria**  I  Admit that I closed my Pull request ,But I have problem that **@Anshika-Jain02**  has fully copied my code and now she is creating issue   with  the same code as of my mine,I know that syntax and variables can be same in github action but How can My  github action Customized greeting Message and Her Customized greetting message is same thats not possible.This means She is Just Copying my code and  taking my issue ,I think this type of practices should not be supported by Admins .

**Comment by thestarsahil** [comment]
"Hello @Nishitbaria  and @Anshika-Jain02 

I understand that there seems to be a dispute regarding the similarity of code in the project. It's crucial for us to approach this situation calmly and work towards a resolution that promotes collaboration and a positive coding environment.

Nishitbaria, I appreciate your concerns and the effort you put into your code. 

Anshika-Jain02, I understand your position as well. Open-source projects often involve shared syntax and common programming practices. However, it's important to address any concerns and provide clarity to maintain a healthy and respectful collaboration.

I would encourage both of you to engage in a constructive discussion, sharing your perspectives and evidence if available.

Let's remember that the goal of this project is to foster collaboration, learn from each other, and create something meaningful together. It's an opportunity for growth and improvement.

Thank you for your understanding and cooperation. Let's work together to find a resolution that benefits the project and the entire coding community.  

@Nishitbaria @Anshika-Jain02  Both Assigned !

**Comment by Nishitbaria** [comment]
It's My Pull  request code check here : 
![image](https://github.com/Counselllor/Counsellor-Web/assets/85815172/2a343332-d881-4cea-9e05-8dd70fd1886f)



It's @Anshika-Jain02  code change Checkout here : 
![image](https://github.com/Counselllor/Counsellor-Web/assets/85815172/36e3c075-7610-4186-8ee0-1815112f57fd)



Now Tell me @thestarsahil  Its copied or not I am not replying to anything after this

**Comment by thestarsahil** [comment]
@Nishitbaria We already Saw this !
Might Happens Relax !

**Comment by Nishitbaria** [comment]
@Anshika-Jain02  according to the rule, the issue is first assigned to me so I am creating my pull request Again
